android/src/fdroid/play/listings/zh-TW/full-description.txt

```diff
@@ -13,7 +13,7 @@
 • 等高線、海拔高度、山峰與谷地
 • 步驟式的行走、單車與實驗性的語言汽車導航
 • 快速的離線搜尋
-• 書籤輸出與匯入 KML/KMZ 格式（GPX 開發中）
+• 書籤輸出與匯入 KML/KMZ/GPX 格式
```

Comment:
Is it better to replace `KML/KMZ/GPX` everywhere to `KML, KMZ, GPX` for SEO purposes?